aspnet/fundamentals/localization.rst

```diff
@@ -0,0 +1,337 @@
+ASP.NET Localization
+====================
+
+By `Tom Archer`_
+
+The ASP.NET 5 localization support provides middleware for establishing the correct culture and UI culture on the thread based on the request, and mechanisms for accessing localized content based on the current culture. In this article, you'll learn how to enable localization and establish the culture using both built-in culture providers. You'll also see how to implement your own custom culture provider.
+
+Enabling localization and establishing culture
+-----------------------------------------------
+
+To enable localization, add the request localization middleware to the request pipeline from the ``Startup`` class with a call to the ``IApplicationBuilder.UseRequestLocalization`` extension method. The ``IApplicationBuilder.UseRequestLocalization`` extension method has two overloaded versions: a parameter-less version that instantiates and uses a default ``RequestLocalizationOptions`` object and a version that allows you to specify a ``RequestLocalizationOptions`` object as a parameter.
+
+.. code-block:: c#
+	:emphasize-lines: 5-6
+
+	public void Configure(IApplicationBuilder app, IStringLocalizer<Startup> SR)
+	{
+		...
+
+		var options = new RequestLocalizationOptions();
+		app.UseRequestLocalization(options);
+
+		...
+	}
+
+The ``RequestLocalizationOptions`` class defines a collection property - ``RequestLocalizationOptions.RequestCultureProviders`` - that represents an ordered list of objects that implement the ``IRequestCultureProvider`` interface.
+
+There are three intrinsic implementations of the ``IRequestCultureProvider`` interface - each of which attempts to establish the correct culture and UI culture on the thread based on various aspects of the incoming request:
+
+- ``QueryStringRequestCultureProvider`` attempts to determine the culture based on the query string.
+- ``CookieRequestCultureProvider`` attempts to determine the culture based on a client-side cookie.
+- ``AcceptLanguageHeaderRequestCultureProvider`` attempts to determine the culture based on the Accept-Language value in the request header.
+
+When you instantiate a ``RequestLocalizationOptions`` object, the constructor instantiates all three of the aforementioned culture providers and adds them to the ``RequestLocalizationOptions.RequestCultureProviders`` collection.
```

Comment:
If I don't tell the user what the c'tor does, then the user might think that they have to add these providers.